draft-ietf-masque-h3-datagram.md

```diff
@@ -137,16 +135,23 @@ that has not yet been created, the receiver SHALL either drop that datagram
 silently or buffer it temporarily while awaiting the creation of the
 corresponding stream.
 
+HTTP/3 datagrams MUST NOT be sent associated with streams that do not define
+semantics for HTTP Datagrams. For example, existing HTTP methods GET and POST do
```

Comment:
This first sentence is a bit hard to read, maybe it's the "sent associated" term. Would this alternative also caputre your intention?

> HTTP/3 datagrams MUST NOT be sent on streams without an associated semantic for HTTP Datagrams.